bin/packages/build-worker.js

```diff
@@ -103,9 +103,13 @@ async function buildCSS( file ) {
 		'animations',
 		'z-index',
 	]
-		// Editor styles should be excluded from the default CSS vars output.
+		// Editor and component styles should be excluded from the default CSS vars output.
 		.concat(
-			file.includes( 'common.scss' ) || ! file.includes( 'block-library' )
+			file.includes( 'common.scss' ) ||
+				! (
+					file.includes( 'block-library' ) ||
+					file.includes( 'components' )
+				)
```

Comment:
This is the part in the build process where all the `--wp-admin-theme-color*` [variables](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_default-custom-properties.scss) are prepended to the main stylesheet of each package.

By excluding the `components` package from this step, the built stylesheet of wp-components will **no longer include the admin scheme color vars**.

This is not a breaking change for wp-components itself. However, if a consumer was using the wp-components package in a non-WP context and _also_ relying on it for providing the `--wp-admin-theme-color*` values that they use in their _own custom components_, then it will be breaking.

Why not load Blueberry scheme `--wp-admin-theme-color*`s for these consumers then? Because setting unscoped `:root` CSS variables that _can differ across multiple wp packages_ is not a good idea. It actually makes things very unpredictable due to load order. So I think we should avoid this practice whenever possible.

(This change will be documented in a Dev Note.)